setupTests.js

```diff
@@ -3,3 +3,15 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 window.HTMLElement.prototype.scrollIntoView = () => {};
+
+/* ---------------------------------------------------------------------------------------------
+ * Below is hack to allow testing of PointerEvent as used in github package
+ * https://github.com/pmndrs/use-gesture
+ ---------------------------------------------------------------------------------------------*/
+/* eslint-disable no-console */
+if (window.PointerEvent) {
+  console.error('ERROR: patching PointerEvent is no longer necessary');
+} else {
+  // console.log('Setup: patching PointerEvent');
```

Comment:
Commented out console log? 🤔

Reply:
I removed the comment on the console log... I was trying to avoid the output of  ```at Object.<anonymous> (setupTests.js:15:11) ``` when the test run and thought it would be good to log it when this hocus-pocus was no longer needed.